File: fibonacci.py
# fibonacci triangle
def fib_triangle(n):
    triangle = []
    for i in range(n):
        row = []
        for j in range(i+1):
            if j == 0 or j == i:
                row.append(1)
            else:
                row.append(triangle[i-1][j-1] + triangle[i-1][j])
        triangle.append(row)
    return triangle
def print_fib_triangle(triangle):
    for row in triangle:
        print(" ".join(map(str, row)))
print("Enter the number of terms: ", end='')
n = int(input())
fibonacci_triangle=fib_triangle(n)
print_fib_triangle(fibonacci_triangle)

File: fibonacci.cpp
#include<iostream>
using namespace std;
int main()
{
    int n, a = 0, b = 1, c;
    cout << "Enter the number of terms: ";
    cin >> n;
    cout << "Triangle Fibonacci: " << endl;
    for (int i = 1; i <= n; ++i)
    {
        cout << b << "\t";
        for (int j = 1; j < i; ++j)
        {
            c = a + b;
            cout << c << "\t";
            a = b;
            b = c;
        }
        cout << endl;
    }
    return 0;
}